cranelift/assembler-x64/meta/src/dsl/format.rs

```diff
@@ -248,35 +256,51 @@ pub enum Location {
     m16,
     m32,
     m64,
+    xmm1,
+    xmm2,
+    xmm3,
+    ymm1,
+    ymm2,
+    ymm3,
+    zmm1,
+    zmm2,
+    zmm3,
+
+    xmm_m128,
```

Comment:
This is the same as `rm128`... I'm not convinced adding these unused variants is necessary at this point, either.

Reply:
I was wanting to be consistent with the manual here:

ADDPD xmm1, xmm2/m128
VADDPD xmm1, xmm2, xmm3/m128

and writing something like this:
```
inst(
            "vaddpd",
            fmt("B", [w(xmm1), r(xmm2), r(xmm_m128)]),
            vex(0x58).length(_128).pp(_66).mmmmm(_OF),
            _64b | compat | sse,
        ),

```
instead of like this:
```
inst(
            "vaddpd",
            fmt("B", [w(xmm1), r(xmm2), r(rm128)]),
            vex(0x58).length(_128).pp(_66).mmmmm(_OF),
            _64b | compat | sse,
        ),
```

I kept the rm128 because I didn't know if it would fly during review that I replaced in in all the rex encoded instructions but using the naming xmm_m128 is more consistent with the manual. What do you think?